python/lsst/ap/verify/testUtils.py

```diff
@@ -45,6 +45,12 @@ class DataTestCase(lsst.utils.tests.TestCase):
     testDataset = 'ap_verify_testdata'
     """The EUPS package name of the dataset to use for testing (`str`).
     """
+    obsPackage = 'obs_lsst'
+    """The obs package associated with ``testDataset`` (`str`).
```

Comment:
Is this approach really necessary? I guess this is future-proofing other subpackages/tests using different data?

Reply:
Pretty much, yes. It seemed silly (and hard to document) to support multiple datasets but require them to all be `obs_lsst`.